Artesian/_Query/VersionedQuery.py

```diff
@@ -144,28 +175,43 @@ def forLastOfMonths(self, start: str, end=None) -> _VersionedQuery:
             self._queryParameters.versionSelectionConfig.versionsRange.dateStart = start
             self._queryParameters.versionSelectionConfig.versionsRange.dateEnd = end
         return self
-    def forLastNVersions(self, lastN: str) -> _VersionedQuery:
+    def forLastNVersions(self, lastN: int) -> _VersionedQuery:
         """ Gets the lastest N timeseries versions that have at least a not-null value .
             
             Args:
-                lastN:  ex. forLastNVersions(2)"""
+                lastN: an int > 0. 
+                Ex.: forLastNVersions(2)
```

Comment:
not sure this works as expected during compilation of Args. Looks like 'Ex.' is another Args.

either move on the 'lastN' line or add an indent tab.